### ISSUE TITLE
Sugesstion : Is possible to add rename pokemon?

### ISSUE THREAD

**Issue by sparkshunter** [Panthere/GoBot#113]
Is possible to add rename pokemon option at "account pokemon"? or add renamed pokemon to list? some example probably "Magikarp(blablabla)". in case if we want to evolve highest IV pokemon without rename it, it's a bit disorganized to find when we have a lot evolved gyarados, where we don't know how many CP we got after it has evolved.

thankyou.

**Comment by Effesus** [comment]
I would agree, this would be very useful when jumping onto the game and knowing which are your pokemon with high IV %.
